backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java

```diff
@@ -319,11 +328,17 @@ public void setGL30 (GL30 gl30) {
 
 	@Override
 	public int getWidth () {
+		if (config.hdpiMode == HdpiMode.Pixels) {
+			return getBackBufferWidth();
+		}

```

Comment:
Minor nitpick but can we have an `if`  `else` structure on all the conditionals?

Reply:
Sure, done